msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md

```diff
@@ -474,22 +465,13 @@ To deserialize the JSON payload, a model object is introduced to get the metadat
 > * Do not use conversation ID as meeting ID.     
 > * Do not use meeting ID from meeting events payload `turncontext.activity.value`. 
       
-The following code shows how to capture the metadata of a meeting that is `MeetingType`, `Title`, `Id`, `JoinUrl`, `StartTime`, and `EndTime` from a meeting start and end event:
+The following code shows how to capture the metadata of a meeting that is `MeetingType`, `Title`, `Id`, `JoinUrl`, `StartTime`, and `EndTime` from a meeting start event:
```

Comment:
As updated by Yunny:
The following code shows how to capture the metadata of a meeting that is `MeetingType`, `Title`, `Id`, `JoinUrl`, `StartTime`, and `EndTime` from a meeting start/end event:
As per MSTP:
```suggestion: The following code shows how to capture the metadata of a meeting that is `MeetingType`, `Title`, `Id`, `JoinUrl`, `StartTime`, and `EndTime` from a meeting start and end event.
```